src/fuzz_introspector/utils.py

```diff
@@ -140,7 +140,11 @@ def demangle_cpp_func(funcname: str) -> str:
 
 
 def demangle_jvm_func(package: str, funcname: str) -> str:
-    return "[%s].%s" % (package, funcname)
+    """Add package class name to uniquly identify jvm functons"""
+    if funcname.startswith("["):
```

Comment:
Does function name ever start with "["? Do you have an example?

Reply:
This is to avoid double assignment of package name. Make it safe when some already processed name will not be processed again.